sklearn/exceptions.py

```diff
@@ -119,9 +119,9 @@ class FitFailedWarning(RuntimeWarning):
     ...         pass
     ...     print(repr(w[-1].message))
     ... # doctest: +NORMALIZE_WHITESPACE
-    FitFailedWarning("Classifier fit failed. The score on this train-test
-    partition for these parameters will be set to 0.000000. Details:
-    \\nValueError('Penalty term must be positive; got (C=-2)',)",)
+    FitFailedWarning('Estimator fit failed. The score on this train-test
```

Comment:
You've included commits from a different PR. Please use branches.

Reply:
Alright, I'll roll back the changes and make a separate PR for this one. Thanks for informing!

Reply:
@jnothman the build fails with HTTP error, can you restart?